### ISSUE TITLE
Add a Pop button

### ISSUE THREAD

**Issue by dcipher00** [dcipher00/Frindi#35]
Add a Pop button in the centre of the bottom Stack navigator so we can use it to flash a card in the future implementation of word for the day.

**Comment by Alexandria** [comment]
Hello! I'd like to take a crack at this.

**Comment by dcipher00** [comment]
@Alexandria sure, go ahead and feel free to raise a PR

**Comment by Alexandria** [comment]
@dcipher00 Do you want this button to point to a blank screen for now?

**Comment by dcipher00** [comment]
@Alexandria yes please create a blank screen for now. The basic idea behind the pop button is that we have to flash a card on top rather than a complete screen navigation so you don't need to create a screen. just create a card with a cross button which would appear once pop button is clicked.
